### PR TITLE
Release v0.5.0

### PR DESCRIPTION
**Pull request by TsumiNa**
@stewu5 Please check https://github.com/yoshida-lab/XenonPy/pull/223/files#diff-3b7c33aad76501d378ca37e20c9c6062356834e40ced388f3e249923cd555ead

Some updates from you not list here because I don't know the issue details. Please fill in the missing parts.

### CODE REVIEW

File: docs/source/changes.rst [hunk 1]
```diff
@@ -5,6 +5,47 @@
 Changes
 =======
 
+v0.5.0
+======
+
+**Breaking change**
+
+* Remove the ``BoxCox`` transform. ( `#222`_ )
```
Comment (stewu5) [inline docs/source/changes.rst:13]: Maybe changed to:
* Remove the ``BoxCox`` transform, and include support for yeo-johnson and box-cox transformation through ``sklearn.preprocessing.PowerTransformer``. ( `#222`_ )

File: docs/source/changes.rst [hunk 2]
```diff
@@ -5,6 +5,47 @@
...
+
+**New features**
+
+* Add new contribution ``iSMD`` by `qi`_. ( `#208`_ )
+* Add classification support for ``Trainer``. ( `#184`_ )
+
```
Comment (stewu5) [inline docs/source/changes.rst:19]: More new features:
* Add ``IQSPR4DF`` to support ``pandas.DataFrame`` input to iQSPR.
* Add ``LayeredFP``, ``PatternFP``, and ``MHFP`` (new rdkit fingerprints).

File: docs/source/changes.rst [hunk 3]
```diff
@@ -5,6 +5,47 @@
...
+v0.5.0
+======
+
+**Breaking change**
+
+* Remove the ``BoxCox`` transform. ( `#222`_ )
+
+**New features**
+
+* Add new contribution ``iSMD`` by `qi`_. ( `#208`_ )
```
Comment (stewu5) [inline docs/source/changes.rst:17]: Shouldn't this be ``ISMD``? (all capitalized)
Also, how about making it more specific:
* Add new contribution ``ISMD`` by `qi`_, a new class of ``BaseProposal`` that allows generation of molecules based on virtual reaction of reactants in a predefined reactant pool. ( `#208`_ )

File: docs/source/changes.rst [hunk 4]
```diff
@@ -5,6 +5,47 @@
...
+* Add classification support for ``Trainer``. ( `#184`_ )
+
+**Enhance**
+
+* ``BaseFeaturizer.transform`` now supports ``pandas.DataFrame`` as input. (??)
```
Comment (stewu5) [inline docs/source/changes.rst:22]: * ``BaseFeaturizer.transform`` now supports ``pandas.DataFrame`` as input where relevant columns for descriptor calculation can be specified through ``target_col``.
* ``BaseDescriptor.transform`` and ``BaseLogLikelihoodSet.log_likelihood`` now automatically check if any group names occur in the input ``pandas.DataFrame`` column names. If not, the entire ``pandas.DataFrame`` will be passed to the corresponding ``BaseFeaturizer`` and ``BaseLogLikelihood``, respectively.

File: docs/source/changes.rst [hunk 5]
```diff
@@ -5,6 +5,47 @@
...
+* Allow using custom elemental information matrix in ``Compositions`` descriptor. ( `#221`_ )
+* Use ``joblib.parallel`` as default parallel backend. ( `#191`_, `#220`_ )
+* ``Splliter.split`` method now support python list as input. ( `#194`_ )
+* Allow user specific index for ``DescriptorHeatmap``. ( `#44`_ )
+* Allow control of number of layers to be extracted in ``FrozenFeaturizer``. ( `#147`_ )
+
```
Comment (stewu5) [inline docs/source/changes.rst:28]: More enhances:
* ``bit_per_entry`` option is added to ``RDKitFP`` and ``AtomPairFP`` to allow control of number of bits to represent one fingerprint entry.
* ``counting`` option is added to ``RDKitFP``, ``AtomPairFP``, ``TopologicalTorsionFP``, ``FCFP`` and ``ECFP`` to support returning counts of each fingerprint entry.
* Column names of ``DescriptorFeature`` is updated to be consistent with the rdkit naming.

**Review by stewu5 — CHANGES_REQUESTED** [review]

File: docs/source/changes.rst (hunk 3 above)
Comment (TsumiNa) [reply docs/source/changes.rst:17]: @qi-zh  I saw Qi use iSMD somewhere. Let's ask Qi for the answer.

File: docs/source/changes.rst (hunk 1 above)
Comment (TsumiNa) [reply docs/source/changes.rst:13]: Thanks! That's exactly what I want. Will replace all the short version with yours

File: docs/source/changes.rst (hunk 2 above)
Comment (TsumiNa) [reply docs/source/changes.rst:19]: Are these two have the issue numbers? If they have, I'd like to put the issue number and links at the end as others did.

File: docs/source/changes.rst (hunk 4 above)
Comment (TsumiNa) [reply docs/source/changes.rst]: comments have been added

File: docs/source/changes.rst (hunk 5 above)
Comment (TsumiNa) [reply docs/source/changes.rst:35]: comments have been added

**Comment by TsumiNa** [comment]
@stewu5 I also changed the README.md in the `contrib` folder, please check it.

File: docs/source/changes.rst (hunk 2 above)
Comment (stewu5) [reply docs/source/changes.rst:22]: I am not sure. The one that I know, I already put... sorry, hard to trace back now.

**Review by stewu5 — APPROVED** [review]

File: docs/source/changes.rst (hunk 3 above)
Comment (stewu5) [reply docs/source/changes.rst]: If we keep it the same as IQSPR, like you make iQSPR => IQSPR now. So, I would say keep it that way too. I remember that you said we should use the convention wherever possible.

File: docs/source/changes.rst (hunk 1 above)
Comment (stewu5) [reply docs/source/changes.rst]: I follow your decision. I kind of think of this as an add-on to the breaking change.

**Pull request merged**